### PR TITLE
fix(Radio): Added default prop options to radio buttons.

### PR DESCRIPTION
**Pull request by Hyperkid123**
Added default prop options of empty array so the component does not throw an error. Has now same behavior like checkbox or select.

### CODE REVIEW

File: packages/pf3-component-mapper/src/form-fields/radio.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  })),
+  isDisabled: PropTypes.bool,
+  isReadOnly: PropTypes.bool,
+  input: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+  }),
+};
+
+RadioGroup.defaultProps = {
+  options: [],
```
Comment (skateman) [inline packages/pf3-component-mapper/src/form-fields/radio.js]: Is this the line I am interested in?

Comment (Hyperkid123) [reply packages/pf3-component-mapper/src/form-fields/radio.js]: yes

Comment (skateman) [reply packages/pf3-component-mapper/src/form-fields/radio.js]: then what's the sugar around it? it seems HUGE for me :confused:

Comment (Hyperkid123) [reply packages/pf3-component-mapper/src/form-fields/radio.js]: You can assign default prop to anonymous function/component. 

Also if you notice all these three components in mappers are very similar. We plan to have "common" components that are shared between packages. This is just a step towards it.

Comment (skateman) [reply packages/pf3-component-mapper/src/form-fields/radio.js]: Okay, it's a little complicated for me, so I think I'm not the right person to review this :disappointed:

**Pull request merged**